stanford.profile

```diff
@@ -133,6 +132,13 @@ function stanford_sites_tasks() {
   }
   else {
     module_enable(array('webauth'));
+    if(stanford_sites_hosted()) {
+      stanford_sites_add_webauth_user(
+        variable_get('stanford_sites_requester_sunetid'),
```

Comment:
Should there be defaults for these variables incase something went wrong and they don't exist?